subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1561,6 +1612,9 @@ std::chrono::steady_clock::time_point GameLogicPluginPrivate::UpdateScoreFiles()
 
   // Output a run summary
   std::ofstream summary(this->logPath + "/summary.yml", std::ios::out);
+  summary << "marsupials:\n";
```

Comment:
If there are no marsupials, this will just print `marsupials:`. Is that okay?

Reply:
I think that is still valid yaml, and would indicate no marsupials.